pypesto/visualize/profiles.py

```diff
@@ -15,50 +15,50 @@ def profiles(
     results: Union[Result, Sequence[Result]],
     ax=None,
     profile_indices: Sequence[int] = None,
-    size: Sequence[float] = (18.5, 6.5),
+    size: tuple[float, float] = (18.5, 6.5),
     reference: Union[ReferencePoint, Sequence[ReferencePoint]] = None,
     colors=None,
```

Comment:
Should this be `colors: Sequence[tuple]` or something similar?

Reply:
"or something similar" :sweat_smile: 
I am not completely sure which types are supported.